.drone.yml

```diff
@@ -1,44 +1,45 @@
-# After any change to this file you MUST regenerate and checkin
-# a .drone.sec even if no secrets were changed. The secrets file is tied
-# to a specific .drone.yml so artifacts will not be uploaded to (OR DOWNLOADED
-# FROM) GCP in following builds without an updated .drone.sec
-#
-# When you are ready to submit a pull request, you must regenerate .drone.sec for the vmware/vic repo:
-#
-# $ drone -s https://ci.vmware.run -t ${DRONE_TOKEN} secure --in "secrets.yml" --out ".drone.sec" --repo vmware/vic --yaml ".drone.yml"
-#
+# After any change to this file you MUST re-sign and checkin the .drone.yml.sig
+# When you are ready to submit a pull request, you must regenerate .drone.yml.sig for the vmware/vic repo:
+# $ drone sign -s https://ci.vcna.io -t ${DRONE_TOKEN} vmware/vic
```

Comment:
Drone0.5 does keep secrets in its DB. 

You can not see the secrets though, just add,remove and list. We should still keep the secrets in our internal repo to version control any changes to secrets and also as a place to document all the secrets. 
I'll send out a separate change to the internal repo for the new secrets script once this is in.